nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>Maybe something like: </p><p>"StatsForecast and MSTL in particular are good benchmarking models for peak detection. However, it might be useful to explore further and newer forecasting algorithms. We have seen particularly good results with a neural network model called NHits from Nixtla's NeuralForecast library. </p><p> </p><p>Learn how to predict ERCOT demand peaks with our deep-learning N-HiTS model and the NeuralForecast library in <a href="https://nixtla.github.io/neuralforecast/examples/electricitypeakforecasting.html" target="_blank">this tutorial</a>." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done</p>